### PR TITLE
Add Site.is_published

### PR DESCRIPTION
**Pull request by blaix**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by blaix** [comment]
ping @RayeN since we talked about this in slack

File: yolapy/models/site.py [hunk 1]
```diff
@@ -8,6 +9,7 @@ class Site(object):
 
     def __init__(self, **kwargs):
         self._fields = kwargs
+        self.publishing_data = kwargs.pop('publishing_data', {})
```
Comment (RayeN) [inline yolapy/models/site.py]: publishing_data should always be present, but it might be `None`.

Comment (blaix) [inline yolapy/models/site.py]: Thanks, fixed

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
:+1: (in case tests pass)

**Pull request merged**